lldb/tools/driver/CMakeLists.txt

```diff
@@ -11,13 +11,23 @@ if(APPLE)
   set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -Wl,-sectcreate,__TEXT,__info_plist,${CMAKE_CURRENT_BINARY_DIR}/lldb-Info.plist")
 endif()
 
+## BEGIN SWIFT
+if (NOT CMAKE_SYSTEM_NAME MATCHES "Darwin|Windows")
+  set(option_set_rpaths BUILD_RPATH "$ORIGIN/../lib" INSTALL_RPATH "$ORIGIN/../lib")
+endif()
+## END SWIFT
```

Comment:
I don't really understand this part, can you explain why this is necessary? I would expect that the driver only needs to find libLLDB and then libLLDB should have the necessary RPATHs to find the swift stuff.

Reply:
@JDevlieghere, the issue is the seemingly buggy behavior of `llvm_setup_rpath()` which is triggered when any LLVM project is built separately, whether compiler-rt upstream or lldb in this Swift fork. Modifying this upstream would require looking into why `llvm_setup_rpath()` adds `LLVM_LIBRARY_DIR` to the runpath, ie finding out who depends on that behavior and seeing if it can be changed.

I am not interested in looking into all that and am just trying to fix this narrow issue for the Swift build, that is triggered by the separate lldb build here.